### ISSUE TITLE
Add Font File to APPServers

### ISSUE THREAD

**Issue by Nadin-Nader** [cityofaustin/atd-data-tech#3191]
### Font file is needed in order for the Barcode to appear on the forms
Issue is related to #1893

**Font file needs to be added to** C:\Windows\Fonts path and possibly C:\Program Files\CSDC\WebServer\webapps\AMANDA5\images\report\font
On servers (coaamandabodev01, coaamandabotst, coaamandabotst2

Request for CTM: SCTASK0077368

**Comment by Nadin-Nader** [comment]
Chat with Surya:

[6/26 8:15 AM] Nader, Nadin
    Good Morning. Sorry i didnt get back to you sooner about the forms. Ok so i had Sree test it and she said that there is a file that needs to go on the appserver in order for the Barcodes to come out instead of just numbers, also one of the column names is cut off and needs adjusting. I have asked Sree to fix theses so as of right now i guess, please hold off on deploying. How would we go about putting a file in the appp server?
​[6/26 8:20 AM] Rathanlal, Surya
    Hmm.. on the Amanda BO server? I think Jenice or Gita would have a better idea
​[6/26 8:21 AM] Rathanlal, Surya
    So I’ll close the current task and you can open a new one when it is ready for PROD.. is that good?
​[6/26 8:21 AM] Nader, Nadin
    yes please. sorry for wasting your time on this 
​[6/26 8:23 AM] Rathanlal, Surya
    No worries Nate.. I can totally understand and glad it is being worked on in DEV and not when moved to PROD 😊
​[6/26 8:29 AM] Nader, Nadin
    me too (smile) I will get with Gita or Jenice to find out whats the best approach while Sree fixes the heading issue
​[6/26 11:36 AM] Nader, Nadin
    Hi
​[6/26 11:37 AM] Nader, Nadin
    ok so i talked to Gita and she said if you have the permissions to add the font file to the APP server, to go ahead and ask you to do it so that we can go ahead and get the forms deployed. I have the task number
[6/26 11:42 AM] Rathanlal, Surya
    should it be placed in a particular directory?
​[6/26 11:43 AM] Nader, Nadin
    i honestly do not know. i sent sreelatha a message asking her the path or if its just on the server
​[6/26 11:44 AM] Rathanlal, Surya
    Ok let me know.. as to access that file she must be using a path and where exactly on the server
​[6/26 11:52 AM] Nader, Nadin
    Yeah. i think she went to lunch. once she gets back i will let you know. Should i assign the task to you?
​[6/26 11:53 AM] Rathanlal, Surya
    Ok this time as I was working on it.. but usually who is on call does the deployments
​[6/26 11:54 AM] Rathanlal, Surya
    But I need more information on the path..
​[6/26 1:34 PM] Nader, Nadin
    
ok so on DEV it needs to go in C:\Program Files\CSDC\WebServer\webapps\AMANDA5\images\report\font and im guessing it goes on the same i the others
​[6/26 1:36 PM] Nader, Nadin
    I added it to the task, I will assign you to the task (sCTASK0077368) once that is done i will create the other for the deployment of the forms
​[6/26 1:38 PM] Rathanlal, Surya
    perfect.. that is what I needed to know!
​[6/26 1:40 PM] Nader, Nadin
    (party)yay. ok i assigned the task to you once you add the file just go ahead and close it and i will have Sree test. Once it goes well i will create the new task for the deployment of the 5 forms. ty and so sorry for the confusion and back and forth
(1 liked)
[11:26 AM] Nader, Nadin
    Hi Surya. Hope you had a great weekend.
​[11:26 AM] Nader, Nadin
    do you have a sec?
​[11:26 AM] Rathanlal, Surya
    Hi Nate.. sure
​[11:30 AM] Nader, Nadin
    So that font file deployment supposedly has not worked.
​[11:31 AM] Rathanlal, Surya
    ok.. what needs to be done?
​[11:31 AM] Nader, Nadin
    They were asking to see if it was because the server had not restarted. Do you know when the server was restarted?
​[11:32 AM] Rathanlal, Surya
    hmm.. not sure. If the dev server needs to be restarted, would have to check with the team and do it
​[11:33 AM] Nader, Nadin
    
Sree asked to see if you could put it in C:\Windows\Fonts on the  same servers (coaamandabodev01, coaamandabotst, coaamandabotst2.) 
​[11:33 AM] Nader, Nadin
    yeah i told her any restarting had to get approval from the team, Gita and Jenice

**Comment by Nadin-Nader** [comment]
Conversation continued:

[11:26 AM] Nader, Nadin
    Hi Surya. Hope you had a great weekend.
​[11:26 AM] Nader, Nadin
    do you have a sec?
​[11:26 AM] Rathanlal, Surya
    Hi Nate.. sure
​[11:30 AM] Nader, Nadin
    So that font file deployment supposedly has not worked.
​[11:31 AM] Rathanlal, Surya
    ok.. what needs to be done?
​[11:31 AM] Nader, Nadin
    They were asking to see if it was because the server had not restarted. Do you know when the server was restarted?
[11:32 AM] Rathanlal, Surya
    hmm.. not sure. If the dev server needs to be restarted, would have to check with the team and do it
​[11:33 AM] Nader, Nadin
    
Sree asked to see if you could put it in C:\Windows\Fonts on the  same servers (coaamandabodev01, coaamandabotst, coaamandabotst2.) 
​[11:33 AM] Nader, Nadin
    yeah i told her any restarting had to get approval from the team, Gita and Jenice
​[11:33 AM] Rathanlal, Surya
    Can we please do one environment at once.. lets try DEV and if it works we can move it to TEST. 
​[11:35 AM] Nader, Nadin
    if you do put them in that new path...the server may need to be restarted. yeah lets do DEV first...i dont want to mess naything up
​[11:37 AM] Rathanlal, Surya
    
So.. is the plan to put the files in the C:\Windows\Fonts path and restart the server or just restart the server to see if the already existing font files worked?
​[11:38 AM] Nader, Nadin
    I would put the file in both paths lol and then restart if need be
​[11:39 AM] Rathanlal, Surya
    ok.. will communicate with the team and have the server restarted
​[11:40 AM] Nader, Nadin
    ok thank you so much just let me know when its done so i can have them test. 
**​[12:00 PM] Rathanlal, Surya
    The font have been placed in the new path and the services have been restarted**

**Comment by Nadin-Nader** [comment]
###**Status:**
Currently deploying files to DEV on the specified paths: C:\Program Files\CSDC\WebServer\webapps\AMANDA5\images\report\font  and  C:\Windows\Fonts on the same servers (coaamandabodev01, coaamandabotst, coaamandabotst2.)
Also restarting Services

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan can you please test to see if the files are on Dev and if the barcode comes out? Once approved Surya will do the same for TEST. Then we can deploy to PROD once TEST is approved.

**Comment by TracyLinder** [comment]
@Nadin-Nader two things:
1. Can you please update the pipeline?
2. Which issue is this related to? 

Thanks!

**Comment by Nadin-Nader** [comment]
@TracyLinder i updated this one and #1893 to show the relationship, pipeline and dependency. :)

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan  @TracyLinder  Surya will be deploying to TEST app servers then once approved to the PROD ones. Once this is done we will be able to deploy the forms on #1893

**Comment by Nadin-Nader** [comment]
Font File has been deployed to TEST. Sree will be checking to see if it works, if so it will be deployed to PROD. 
Once this is complete, the templates (#1893) can be deployed to DEV TEST and PROD.

**Comment by Nadin-Nader** [comment]
Font Files have been deployed to all three environments. Closing card.
